shark/iree_utils/compile_utils.py

```diff
@@ -343,7 +343,7 @@ def get_results(compiled_vm, input, config, frontend="torch"):
         res = np.array(data, dtype=object)
         return np.copy(res)
```

Comment:
I think we can make the same changes here and above as well.

Reply:
Because this change isn't fully understood yet I don't want to update the above just yet. A future update can unify these cases.